www/site/content/docs/reference/config.md

```diff
@@ -0,0 +1,262 @@
++++
+title = "configuration"
+linkTitle = "configuration"
+[menu]
+  [menu.docs]
+    parent = "Reference"
+    weight = "10"
++++
+
+# Configuration
+
+Chef Workstation App and `chef-run` configuration is managed in `config.toml`.
+Chef Workstation will create a `config.toml` the first time you use `chef-run`, if one does not already exist.
+To configure of other tools, see their respective pages found in the [ChefDK documentation](https://docs.chef.io/about_chefdk.html).
+
+## Default location
+
+Chef Workstation looks for the `config.toml` in a default location.
+
+### Windows
+
+* Powershell: `$env:USERPROFILE\.chef-workstation\config.toml`
+* cmd.exe: `%USERPROFILE%\.chef-workstation\config.toml`
+
+### Linux and Mac
+
+`/home/$USER/.chef-workstation/config.toml`
+
+## Settings
+
+### [telemetry]
+
+Configure telemetry behaviors for Chef Workstation components.
+
+#### Example
+
+```toml
+[telemetry]
+enable=true
+dev=false
+```
+
+#### Option: `enable`
```

Comment:
Remove backticks from `enable` (and others)

Reply:
Are "others" consider the other Options?